torch/_inductor/select_algorithm.py

```diff
@@ -849,47 +846,48 @@ def benchmark_in_current_process(choice):
             torch.cuda.synchronize()  # shake out any CUDA errors
             return result
 
-        def benchmark_in_sub_process(choice):
-            # only benchmark triton kernel in sub process for now.
-            # ATen/Extern kernel are still benchmarked in the current process.
-            if isinstance(choice, ExternKernelCaller):
-                return benchmark_in_current_process(choice)
-
+        def benchmark_in_current_process(options):
+            timings = {}
+            for choice in options:
```

Comment:
nit: should we uniformly call it 'choice' and 'choices'?

Reply:
@shunting314, sure I'll change. That's what I had initially, but as it turns out this is a nested function and the outer method _also_ has a param called 'choices'. So I didn't love the ambiguity. It would be easy to misspell 'options', for example, and be none-the-wiser because we'd silently reference the 'options' from the outer scope. I guess this is one of the reasons why I personally don't love nested functions.